app/gui/src/dashboard/modules/payments/components/PlanSelector/components/Card.tsx

```diff
@@ -25,61 +24,37 @@ interface CreateCheckoutSessionMutationParams {
   readonly period: PlanBillingPeriod
 }
 
-const TEXT_ID_FOR_BILLING_PERIOD: Record<PlanBillingPeriod, TextId & `billingPeriod${string}`> = {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  1: 'billingPeriodMonthly',
-  // eslint-disable-next-line @typescript-eslint/naming-convention, @typescript-eslint/no-magic-numbers
-  12: 'billingPeriodAnnually',
-}
-
 /** The component for a plan. */
-export interface PropsForPlan<PlanVariant extends Plan> {
-  readonly pricing: TextId & `${PlanVariant}PlanPricing`
-  readonly features: TextId & `${PlanVariant}PlanFeatures`
-  readonly title: TextId & `${PlanVariant}PlanName`
-  readonly subtitle: TextId & `${PlanVariant}PlanSubtitle`
+export interface PropsForPlan {
   readonly submitButton: (props: SubscribeButtonProps) => React.ReactNode
   readonly elevated?: boolean
 }
 
-const PROPS_FOR_PLAN: {
-  readonly [PlanVariant in Plan]: PropsForPlan<PlanVariant>
-} = {
-  free: {
-    pricing: 'freePlanPricing',
```

Comment:
we might want to remove the associated strings in `english.json` - they should be all close together so it should be easy enough to remove. and remember there is nothing to worry about removing too much as `getText` is typesafe, so if you remove too much then typechecking will fail